docs/spec_draft.md

```diff
@@ -388,6 +389,50 @@ dimensions `k` in `operand`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.case
+
+### Semantics
+
+Returns the result of exactly 1 function from `branches` depending on the value
+of `index`. Formally, if $0 \le$ `index` $\lt$ `N-1`, result of executing
+`branches[index]` is returned, else, result of executing `branches[N-1]` is
+returned.
```

Comment:
How about "Produces the output of"? (Similarly for other uses of "return" and "result" in this paragraph). Related discussion: https://github.com/openxla/stablehlo/pull/325/files#r996106105.